packages/snippet-bot/src/snippet-bot.ts

```diff
@@ -286,6 +286,17 @@ async function scanPullRequest(
   const owner = context.payload.repository.owner.login;
   const repo = context.payload.repository.name;
 
+  // If the head repo is null, we can not proceed.
```

Comment:
I suspect, in the `app.on('pull_request')` style code somewhere in this project, you could check for `pull_request.state` (or something like that), check if it's closed, and just short circuit all of the code paths if this PR is closed far before getting here.

Reply:
I think that's another good check. I created #1183